### PR TITLE
Feature/add schedule 5133

### PR DESCRIPTION
**Pull request by AndrienkoMS**

### CODE REVIEW

File: syndicate/connection/eventbridge_schedule_connection.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+from botocore.exceptions import ClientError
+
+from syndicate.commons.log_helper import get_logger
+from syndicate.connection.helper import apply_methods_decorator, retry
+
+_LOG = get_logger('syndicate.connection.eventbridge_schedule_connection')
+
+
+@apply_methods_decorator(retry)
+class EventBridgeScheduleConnection(object):
```
Comment (bohdan-onsha) [inline syndicate/connection/eventbridge_schedule_connection.py:11]: Service is called AWS EventBridge Schedule<ins>r</ins>
Please rename the file and respective classes

File: syndicate/core/resources/eventbridge_schedule_resource.py [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+    def __init__(self, eventbridge_conn):
+        self.connection = eventbridge_conn
+
+    def create_schedule(self, args):
+        return self.create_pool(self._create_schedule_from_meta, args)
+
+    @unpack_kwargs
+    def _create_schedule_from_meta(self, name, meta):
+        _LOG.debug('Creating schedule %s', name)
```
Comment (bohdan-onsha) [inline syndicate/core/resources/eventbridge_schedule_resource.py:88]: Don't mess several types of string formatting in a single place. 
Just use the most frequently used in the project (f-strings)

Comment (AndrienkoMS) [reply syndicate/core/resources/eventbridge_schedule_resource.py:88]: done

File: syndicate/connection/eventbridge_schedule_connection.py (hunk 1 above)
Comment (AndrienkoMS) [reply syndicate/connection/eventbridge_schedule_connection.py:11]: done

File: setup.py [hunk 3]
```diff
@@ -27,6 +27,7 @@
 
 setup(
     name='aws-syndicate',
+    version='1.4.0',
     version='1.3.3',
```
[the remark this answers is not in the document]
Comment (AndrienkoMS) [reply setup.py:31]: done

File: syndicate/connection/__init__.py [hunk 4]
```diff
@@ -166,6 +167,13 @@ def firehose(self, region=None):
             credentials['region'] = region
         return FirehoseConnection(**credentials)
 
+    @lru_cache(maxsize=None)
+    def eventbridge_schedule(self, region=None):
```
Comment (bohdan-onsha) [inline syndicate/connection/__init__.py:171]: .

File: syndicate/core/resources/eventbridge_scheduler_resource.py [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+
+        :type name: str
+        :type meta: dict
+        """
+        check_params = meta['schedule_content']
+        check_params['name'] = name
+        group_name = check_params.get('group_name')
+        response = self.connection.describe_scheduler(name, group_name)
+        if not response:
+            raise AssertionError('{0} schedule does not exist.'.format(name))
```
Comment (bohdan-onsha) [inline syndicate/core/resources/eventbridge_scheduler_resource.py:121]: f-strings to use

File: syndicate/connection/__init__.py (hunk 4 above)
Comment (AndrienkoMS) [reply syndicate/connection/__init__.py:171]: done

**Review by bohdan-onsha — APPROVED** [review]

**Pull request merged**